src/drivers/batt_smbus/batt_smbus.cpp

```diff
@@ -89,6 +89,9 @@ void BATT_SMBUS::RunImpl()
 	// Read data from sensor.
 	battery_status_s new_report = {};
 
+	// TODO(hyonlim): this driver should support multiple SMBUS going forward.
+	new_report.id = 1;
```

Comment:
I don't think this is needed for smart batteries. This ID is used for dumb batteries to properly match and apply the `BAT_*` parameters, which are not used for smart batteries. For now I think we should just pick a special number for ID to indicate it's a smart battery (say 99) just to prevent confusion.

Reply:
The TODO for multiple SMBUS drivers is also a separate problem altogether and outside of the scope of these changes.

Reply:
If we don't do this, QGroundcontrol rejects discharged_mah message. Please check.
That's why I set this as 1.

Reply:
Okay, let's just leave it for now